projects/packages/forms/src/blocks/contact-form/block.json

```diff
@@ -21,6 +21,11 @@
 		},
 		"align": [ "wide", "full" ]
 	},
-	"attributes": {},
+	"attributes": {
+		"selectedStepClientId": {
```

Comment:
Doesn't seem ideal to store what's essentially _state_ in attributes which get stored in block markup. Any way to avoid this?